### PR TITLE
Make logging in admission controllers consistent with controller-manager

### PR DESCRIPTION
**Pull request by pmorie**
I noticed while doing triage on bugs the other day that there are a few nits in various admission controllers re: logging that I thought should be fixed.

### CODE REVIEW

**Comment by staebler** [comment]
LGTM

File: plugin/pkg/admission/broker/authsarcheck/admission.go [hunk 1]
```diff
@@ -74,23 +74,23 @@ func (s *sarcheck) Admit(a admission.Attributes) error {
 	if a.GetResource().Group != servicecatalog.GroupName || a.GetResource().GroupResource() != servicecatalog.Resource("clusterservicebrokers") {
 		return nil
 	}
-	clusterClusterServiceBroker, ok := a.GetObject().(*servicecatalog.ClusterServiceBroker)
```
Comment (MHBauer) [inline plugin/pkg/admission/broker/authsarcheck/admission.go]: the name, lol.

File: plugin/pkg/admission/namespace/lifecycle/admission.go [hunk 2]
```diff
@@ -129,7 +129,7 @@ func (l *lifecycle) Admit(a admission.Attributes) error {
 			return nil
 		}
 
-		return admission.NewForbidden(a, fmt.Errorf("unable to create new content in namespace %s because it is being terminated", a.GetNamespace()))
```
Comment (MHBauer) [inline plugin/pkg/admission/namespace/lifecycle/admission.go]: @duglin should like this. `%q`all the things.

File: plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go [hunk 3]
```diff
@@ -98,30 +98,30 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
 
 	plans, err := d.getClusterServicePlansByClusterServiceClassName(sc.Name)
 	if err != nil {
-		msg := fmt.Sprintf("Error listing plans for service class (K8S: %v ExternalName: %v) - retry and specify desired ClusterServicePlan", sc.Name, instance.Spec.ClusterServiceClassExternalName)
-		glog.V(4).Info(msg)
+		msg := fmt.Sprintf("Error listing ClusterServicePlans for ClusterServiceClass (K8S: %v ExternalName: %v) - retry and specify desired ClusterServicePlan", sc.Name, instance.Spec.ClusterServiceClassExternalName)
+		glog.V(4).Infof(`ServiceInstance "%s/%s": %s`, instance.Namespace, instance.Name, msg)
```
Comment (MHBauer) [inline plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go]: should this be part of the log messaeg only or incorporated into the msg that gets set?

File: plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go [hunk 4]
```diff
@@ -98,30 +98,30 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
...
 		return admission.NewForbidden(a, errors.New(msg))
 	}
 
 	// check if there were any service plans
 	// TODO: in combination with not allowing classes with no plans, this should be impossible
 	if len(plans) <= 0 {
-		msg := fmt.Sprintf("no plans found at all for service class %q", instance.Spec.ClusterServiceClassExternalName)
-		glog.V(4).Info(msg)
+		msg := fmt.Sprintf("no ClusterServicePlans found at all for ClusterServiceClass %q", instance.Spec.ClusterServiceClassExternalName)
+		glog.V(4).Infof(`ServiceInstance "%s/%s": %s`, instance.Namespace, instance.Name, msg)
```
Comment (MHBauer) [inline plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go]: differing log v message content

File: plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go [hunk 5]
```diff
@@ -98,30 +98,30 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
...
 		return admission.NewForbidden(a, errors.New(msg))
 	}
 
 	// check if more than one service plan was specified and error
 	if len(plans) > 1 {
 		msg := fmt.Sprintf("ServiceClass %q has more than one plan, PlanName must be specified", instance.Spec.ClusterServiceClassExternalName)
-		glog.V(4).Info(msg)
+		glog.V(4).Infof(`ServiceInstance "%s/%s": %s`, instance.Namespace, instance.Name, msg)
```
Comment (MHBauer) [inline plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go]: differing log v message content

File: plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go (hunk 3 above)
Comment (n3wscott) [reply plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go]: Use a `PrettyContextBuilder` for fun and profit

**Comment by n3wscott** [comment]
LGTM except it would be nice to use the the PrettyContextBuilder to help remove some of the redundant logging effort. Some places it does not help because the function is just not long enough... This could be one of those places.

**Pull request merged**